### PR TITLE
[15.0][ADD] account_payment_order_tier_validation: new module to add tier validation to payment orders

### PR DESCRIPTION
**Pull request by alan196**

### CODE REVIEW

**Review by bosd — APPROVED** [review]
Code review LGTM

**Comment by GasparJarsa** [comment]
Functional review.

File: account_payment_order_tier_validation/views/account_payment_order_view.xml
Comment (GasparJarsa) [inline account_payment_order_tier_validation/views/account_payment_order_view.xml:1]: Functional review.

**Review by GasparJarsa — APPROVED** [review]
Functional review.

**Pull request closed without merging**